src/tests/test_fields.py

```diff
@@ -93,7 +94,7 @@ def test_bytes():
 
     # Hex encoding doesn't accept non-hex inputs
     with pytest.raises(TypeError):
-        hex_bytes_field.adapt(u'hijklmnop')
+        hex_bytes_field.adapt(u'hijklmnopq')
```

Comment:
using `codecs` to do encode/decode throws a different error -> If it tries to decode this as hex, it immediately fails because hex chars are 2 digits each (odd vs even length). I wanted it to fail more fundamentally, so I added char to be even.